.github/workflows/ghcr.yml

```diff
@@ -38,3 +38,10 @@ jobs:
         run: |
           ORG_NAME=$(echo "${{ github.repository }}" | tr '[A-Z]' '[a-z]' | cut -d '/' -f 1)
           ./containers/build.sh ${{ matrix.image }} $ORG_NAME --push
+
+  docker_build_success:
```

Comment:
this may miss cancel and skip cases
here is extended version https://github.com/google/python-fire/pull/435/files